config/license_finder.yml

```diff
@@ -0,0 +1,6 @@
+---
+project_name: Porta
+dependencies_file_dir: './doc/licenses'
```

Comment:
btw setting also the dependency decisions file worked for me
```
🐚 cat config/license_finder.yml 
---
project_name: Porta
dependencies_file_dir: './doc/licenses'
decisions_file: './doc/licenses/decisions.yml'
enabled_package_managers:
  - bundler
  - yarn
```
When I ran some commands, they were applied to the specified decisions file. If you want to have its path changed, the above worked.

Reply:
Yes, problem is with Github in this case, it complains about unsolvable conflicts in `dependency_decisions.yaml` even though it's just a file name change. Moreover the default location is `doc/dependency_decisions.yml` so it's better not to change it after all.

Reply:
IMO, it's a strange location. It's more for `config` something but I'm fine with keeping it where it is.